accepted/0000-composer-module.md

```diff
@@ -0,0 +1,125 @@
+# Composer Module
+
+## Summary
+
+Add new library `@orchestrator/composer` that will allow to compose configuration for orchestrator visually via drag-n-drop.
+
+## Motivation
+
+This will allow users to configure their UI visually and generate JSON from that for future use.
+
+## Detailed Explanation
+
+It should expose component `<orc-composer>` that will render all UI controls:
+
+- `<orc-composer-components>` - will render list of all available components ready for dragging
+- `<orc-composer-canvas>` - will render area for dropping components and configure them
+- `<orc-composer-preview>` - will render currently configured components in live preview
```

Comment:
Also I would add 2 more components: 

- `<orc-composer-errors>` - will render list of errors that were while rendering
- `<orc-composer-json>` - will render result json, aslo it should be possible to edit the whole config via this json. So when user clicks to download button, we can show this result and copy everything to clipboard.

Reply:
Both of them can be part of `<orc-composer-preview>` but named differently and used internally in it.